IA shuffle and edits (#13891)
PrefectHQ/prefect: docs/3.0rc/deploy/work-pools/control-runs.mdx

You can also run your flow through the Prefect UI: https://app.prefect.cloud/account/
abc/workspace/123/deployments/deployment/xyz
```

Navigate to your Prefect Cloud UI and view your new deployment.
Click the **Run** button to trigger a run of your deployment.

Because you configured this deployment with a Prefect Managed work pool, Prefect Cloud will run your flow on your behalf.

You can view the logs in the UI.

### Schedule a deployment run

You're ready to submit a flow-run to the work pool.
Run the deployment from the CLI (or the UI):

```bash
prefect deployment run 'get_repo_info/my-deployment'
```

Prefect Managed work pools are a great way to get started with Prefect.  
See the [Managed Execution guide](/3.0rc/guides/managed-execution/) for more details.

For more control over the infrastructure that your flows run on, we recommend Prefect Cloud's push work pools.

## Push work pools with automatic infrastructure provisioning

Serverless push work pools scale infinitely and provide more configuration options than Prefect Managed work pools.

Prefect provides push work pools for AWS ECS on Fargate, Azure Container Instances, Google Cloud Run, and Modal.

> Using a push work pool requires sufficient permissions on your cloud provider account.

Reviewer: ```suggestion
Push work pools require a cloud provider account with certain permissions.
```
